appveyor.yml

```diff
@@ -1,4 +1,4 @@
-os: Windows Server 2012 R2
+os: Visual Studio 2015
```

Comment:
Why does the VS version get listed in the OS label?  Is that just how appveyor works?